### DISCUSSION TITLE
Ordering groups in the tablist

### DISCUSSION THREAD

**Discussion by Strahilchu** [Zrips/CMI#5948] [Enhancement]
Basically reopening #931 from 3 years ago! I strongly believe it should be reconsidered :)

An ordered Tablist is due!

Explanation: When you hold Tab to view the player list, admins should be on top, then mods>donators and members on bottom, or any way the owner configures it.

This is still not an option in CMI and I find it odd

**Comment by CloudeLecaw** [comment]
Maybe this is one reason why it was landes in discussion like the 10 others that suggested this.

Well bit true it one of the may most wanted features ;)

**Comment by Strahilchu** [comment]
I think it makes sense if CMI is to have a Tablist at all it should be done well! and it's great, this is the only thing that it lacks that I can think of

**Comment by CloudeLecaw** [comment]
I know like and agree, but u noticed allready... we suggest and want this since over two years now :)

So i dont belive it will come so soon.
